### ISSUE TITLE
ImportError: No module named dlmanager

### ISSUE THREAD

**Issue by alonecoder1337** [parkouss/dlmanager#1]
![screenshot from 2016-10-02 11-36-06](https://cloud.githubusercontent.com/assets/6220530/19019031/9113e552-8894-11e6-8b32-35a386fb4415.png)

**Comment by tarekziade** [comment]
Yeah that bit me too. The project's setup.py installs everything except the actual code. It pissings a packages direcive
